docs/core/extensions/workers.md

```diff
@@ -42,6 +42,17 @@ The preceding `Program` class:
 - Builds an <xref:Microsoft.Extensions.Hosting.IHost> from the builder.
 - Calls `Run` on the `host` instance, which runs the app.
 
+> [!TIP]
+> By default the Worker Service template doesn't enable server garbage collection (GC). All of the scenarios that require long-running services should consider performance implications of this default. To enable server GC, add the `ServerGarbageCollection` node to the project file:
+>
+> ```xml
+> <PropertyGroup>
+>      <ServerGarbageCollection>true</ServerGarbageCollection>
+> </PropertyGroup>
+> ```
+>
```

Comment:
It might not be a bad idea to show the runtime config version too. It's possible someone is looking at this because the  need to tweak an already deployed service. The "examples" link text doesn't really indicate that there are alternatives to editing and recompiling the project.

Reply:
I linked to it in this alert. I didn't want to show that one specifically as this article only discussed the .csproj.